src/assets/labContent.js

```diff
@@ -44,21 +40,19 @@ export default [
       subheader: "How do the appearance of ads differ across individual accounts?",
       bodyText: [
         {
-          body: `Computer programs categorize users based on their demographical characteristics in order to optimize efficiency in many things, including advertising. By incorporating these categorizations into advertising algorithms, advertisers are able to maximize the relevancy of their ads and thus maximize their profit.`
+          body: <text>Computer programs <mark style={{backgroundColor: '#ffedbd', borderRadius: '7px', padding: '0 5px 5px 5px'}}>categorize</mark> users based on their demographical characteristics in order to optimize efficiency in many things, including advertising. Through such categorizations, advertisers are able to <u>maximize the relevancy of their ads and thus maximize their profit.</u></text>
```

Comment:
Can these inlined styles be made into CSS that is inside our larger mainContent.css file? It would help reduce clutter down the line if you want to add more highlighted text sections.